### DISCUSSION TITLE
@riverpod for a provider that depends on a StreamProvider

### DISCUSSION THREAD

**Discussion by julienlebren** [rrousselGit/riverpod#2226] [General]
I have a provider generated with riverpod_generator which depends on a StreamProvider.
This StreamProvider can't be generated by riverpod_generator because it's not supported.
But all the dependencies of my initial provider need to be generated with riverpod_generator to work...
I'm struggling to figure this out. 
Why riverpod_generator does not generate StreamProvider? 
I'm stuck on this 🤯

**Comment by AhmedLSayed9** [comment]
It should be available soon. There's an open PR currently.
